dodiscover/constraint/config.py

```diff
@@ -0,0 +1,20 @@
+from enum import Enum, EnumMeta
+
+
+class MetaEnum(EnumMeta):
```

Comment:
Can we add a note about the purpose of this metaclass?

Reply:
Added a note